frontend/src/Components/VolunteersCard.jsx

```diff
@@ -19,7 +24,7 @@ const VolunteerSearchCard = (props) => {
                         <p className="card-text">{props.volunteer.skills[1]}</p>
                         <p className="card-text">{props.volunteer.skills[2]}</p>
                     </ul>
-                    <h5>Next Event: {props.volunteer.topics[0]} (make this a link)</h5>
+                    <h5>Next Event: {props.volunteer.topic_ids[0]} (make this a link)</h5>
```

Comment:
This will display the event id instead of the event name, for now it's just a place holder since we're planning to have it as a link to the event itself, it's also the reason I have axios and setFeedback to do a network request to have the event